opk.go

```diff
@@ -47,7 +51,7 @@ func (sv *opkSignerVerifier) Sign(ctx context.Context, data []byte) ([]byte, err
 	return pkTokenJSON, nil
 }
 
-func (sv *opkSignerVerifier) Verify(ctx context.Context, data, sig []byte) error {
+func (sv *opkSignerVerifier) Verify(ctx context.Context, data []byte, sig []byte) error {
```

Comment:
Does this pass the linter?

Reply:
Yeah, I caught and fixed this by just noticing it. No linter or compiler warnings which was surprising.

Reply:
Sorry, I was asking about it the other way around. Typically in go if two variables have the same type, you don't need the type twice, so `a, b int` is shorthand, and typically preferred, for `a int, b int`.

Reply:
That explains it! Not sure that I like that, but it takes up less space [6f10d7a](https://github.com/openpubkey/signed-attestation/pull/4/commits/6f10d7ae001abfa95098256e233c37193f5206ac)